src/app/(authenticated)/monitoring/components/monitoring-presence-client.tsx

```
@@ -81,6 +89,42 @@ export default function MonitoringPresenceClient() {
+    return Array.isArray((p as unknown as { attendances?: unknown }).attendances);
+  };
+
+  const statusLabel: Record<"h" | "s" | "i" | "a" | "b", string> = {
+    h: "Hadir",
+    s: "Sakit",
+    i: "Izin",
+    a: "Alpha",
+    b: "Bolos",
+  };
+
+  const filteredStudents = useMemo(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on hasAttendances.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)